lib/Form/Block/Base.php

```diff
@@ -6,7 +6,7 @@
 use MailPoet\Models\ModelValidator;
 use MailPoet\WP\Functions as WPFunctions;
 
-abstract class Base {
+class Base {
```

Comment:
I am not sure about the class name. I mean it was called `Base` because it was extended in all the other block renderers. Now it is not extended any more so we should probably use a better name. 

I am doubtful about the need for this class. Why are all of those methods in a single class?